sdk/storage/AzureStorageBlob/Source/Operation/ResumableOperationQueue.swift

```diff
@@ -105,44 +99,21 @@ internal class ResumableOperationQueue {
                         }
                         operation.internalState = .complete
                     }
-                    self.operations.append(operation)
                 }
                 operationQueue.addOperations(filteredOps, waitUntilFinished: false)
             }
         }
     }
 
-    public func cancel(_ operation: ResumableOperation) {
+    func cancel(_ operation: ResumableOperation) {
         operation.cancel()
     }
 
-    public func clear() {
+    func clear() {
         operationQueue.cancelAllOperations()
-        operations.removeAll()
     }
 
-    internal func removeFromDataStore(_ operation: ResumableOperation) {
-        guard let index = operations.firstIndex(of: operation) else { return }
-        operations.remove(at: index)
-    }
-
-    public func pause(_ operation: ResumableOperation) {
+    func pause(_ operation: ResumableOperation) {
         operation.pause()
-        removeFromDataStore(operation)
-    }
-
-    public func remove(_ operation: ResumableOperation) {
-        removeFromDataStore(operation)
-
-        // Cancel the operation.
-        if !operation.isFinished {
-            operation.cancel()
-        }
-    }
-
-    public func replace(old oldOperation: ResumableOperation, withNew newOperation: ResumableOperation) {
```

Comment:
No longer necessary. Pausing simply cancels the relevant operations. Resuming the paused transfer results in the necessary operations being requeued. The necessary state is tracked in the Transfer object.